qrenderdoc/Windows/ShaderViewer.ui

```diff
@@ -319,6 +319,29 @@
       </property>
      </widget>
     </item>
+    <item>
+     <widget class="QToolButton" name="bookmark">
+      <property name="toolTip">
+       <string>Run or step backwards in execution</string>
```

Comment:
Sorry, I noticed this last time and I would have _sworn_ that I put a comment on it, but I must have been mistaken. This tooltip is not correct for the button.

Reply:
Would this be ok?

```
      <property name="toolTip">
       <string>Add, remove or jump to bookmarks in current file</string>
      </property>
```

Reply:
Yep that sounds good!